Client/Frontend/Browser/Tab.swift

```diff
@@ -317,7 +317,6 @@ class Tab: NSObject {
       }
       let webView = TabWebView(frame: .zero, tab: self, configuration: configuration!, isPrivate: isPrivate)
       webView.delegate = self
-      configuration = nil
```

Comment:
Note: I do NOT know why we ever set the configuration here to `nil`. 
I have removed it as I tested and it has no negative impacts. Also Firefox removed it as well.